src/containers/TermsOfUse/index.tsx

```diff
@@ -0,0 +1,330 @@
+            appropriate and comply with these Terms of Service, remove any and/or all of your submissions, and terminate
+            your account with or without prior notice.
+          </li>
+        </ul>
+        <p>
+          You understand and agree that any liability, loss or damage that occurs as a result of the use of any User
+          Content that you make available or access through your use of the Service is solely your responsibility. The
+          Company is not responsible for any public display or misuse of your User Content. The Company does not, and
+          cannot, pre-screen or monitor all User Content. However, at our discretion, we, or the technology we employ,
+          may monitor and/or record your interactions with the Service.
+        </p>
+
+        <h2 className="TermsOfUse__subtitle">Online Content Disclaimer</h2>
+        <p>
+          Opinions, advice, statements, offers, or other information or content made available through the Service, but
+          not directly by the Company, are those of their respective authors, and should not necessarily be relied upon.
+          Such authors are solely responsible for such content. The Company does not guarantee the accuracy,
+          completeness, or usefulness of any information on the Service and neither does the Company adopt nor endorse,
+          nor is the Company responsible for the accuracy or reliability of any opinion, advice, or statement made by
+          parties other than the Company. The Company takes no responsibility and assumes no liability for any User
+          Content that you or any other user or third party posts or sends over the Service. Under no circumstances will
+          the Company be responsible for any loss or damage resulting from anyone's reliance on information or other
+          content posted on the Service, or transmitted to users. Though the Company strives to enforce these Terms of
+          Use, you may be exposed to User Content that is inaccurate or objectionable. The Company reserves the right,
+          but has no obligation, to monitor the materials posted in the public areas of the service or to limit or deny
+          a user's access to the Service or take other appropriate action if a user violates these Terms of Use or
+          engages in any activity that violates the rights of any person or entity or which we deem unlawful, offensive,
+          abusive, harmful or malicious. The Company shall have the right to remove any such material that in its sole
+          opinion violates, or is alleged to violate, the law or this agreement or which might be offensive, or that
+          might violate the rights, harm, or threaten the safety of users or others. Unauthorized use may result in
+          criminal and/or civil prosecution under the law. If you become aware of misuse of our Service, please contact
```

Comment:
not sure about this - but it probably needs to add the law of which state/country?